Samples/MediaCaptureSampleExtendedControl/BasicCapture.xaml.h

```diff
@@ -0,0 +1,87 @@
+﻿//*********************************************************
+//
+// Copyright (c) Microsoft. All rights reserved.
+// THIS CODE IS PROVIDED *AS IS* WITHOUT WARRANTY OF
+// ANY KIND, EITHER EXPRESS OR IMPLIED, INCLUDING ANY
+// IMPLIED WARRANTIES OF FITNESS FOR A PARTICULAR
+// PURPOSE, MERCHANTABILITY, OR NON-INFRINGEMENT.
+//
+//*********************************************************
+
+//
+// BasicCapture.xaml.h
+// Declaration of the BasicCapture class
+//
+
+#pragma once
+
+#include "pch.h"
+#include "BasicCapture.g.h"
+#include "MainPage.xaml.h"
+
+using namespace Windows::UI::Xaml;
+using namespace Windows::UI::Xaml::Controls;
+using namespace Windows::Graphics::Display;
+using namespace Windows::UI::ViewManagement;
+using namespace Windows::Devices::Enumeration;
+#define VIDEO_FILE_NAME "video.mp4"
+#define PHOTO_FILE_NAME "photo.jpg"
```

Comment:
use const variables, also how is this not L"" and still working

Reply:
No idea. All of the streaming setup/usage is from our existing Camera samples. I can adjust here and test, but it is working.

Reply:
I think that is allowed for Platform::String - [https://docs.microsoft.com/en-us/cpp/cppcx/platform-string-class?view=vs-2019](https://docs.microsoft.com/en-us/cpp/cppcx/platform-string-class?view=vs-2019)


---
In reply to: [373708734](https://github.com/microsoft/Windows-Camera/pull/11#discussion_r373708734) [](ancestors = 373708734)

Reply:
Won't fix.